### DISCUSSION TITLE
利用opencv追蹤手部運動

### DISCUSSION THREAD

**Discussion by ChinChun0220** [1101-GS4719-C/Python-Community-GS4719-C#325] [Ideas]
我記得前幾天有一篇文在討論有個app能夠反映手語至文字，結果沒幾天我的YouTube 就推薦跟我這個影片了（神奇的演算法），是通過opencv （一個實驗課用過的影像分析軟體）來捕捉手部運動，在透過AI學習～
我自己是沒看完，給有興趣的同學參考看看～

感覺除了手語翻譯外還有很多應用可以做！

https://youtu.be/x4eeX7WJIuA

**Comment by ChinChun0220** [comment]
https://www.google.com/amp/s/chtseng.wordpress.com/2020/12/09/%25E6%2589%258B%25E5%258B%25A2%25E5%2581%25B5%25E6%25B8%25AC%25E6%2587%2589%25E7%2594%25A8%25E6%2596%25BC%25E7%258E%25A9game/amp/
大約查了一下，發現目前發展比較成熟的應該是在遊戲上，透過真實的運動來玩遊戲

**Comment by cody910120** [comment]
那以後現場直播靠手語老師就有字幕可以看了ㄟ

**Reply by ChinChun0220** [reply]
直播那種要字幕也不會從手語轉啦😅 語言輸入技術以及很發達了

**Comment by mesakiyuta** [comment]
透過影像去追蹤手部運動，不用特別準備甚麼東西感覺好方便，不知道虛擬偶像(Vtuber)有沒有運用類似的技術

**Reply by ChinChun0220** [reply]
虛擬偶像、vtuber 目前應該還是穿著實體的感測器來偵測動作，不過若能夠變成完全空手偵測感覺很不錯，能減少設備金額的門檻，然更多人投入

**Comment by swerfg03joe** [comment]
利用google開發的mediapipe確實可以快速抓取手部動作，然而他抓取的是手的關節點，所以如果要辨識手勢的話還需要做額外的定義，再來手語翻譯就更加的複雜了，因為手部動作是連續的這部分比較難切開，需要在下點其他功夫，不過方向大致如此

**Comment by BlahFish1006** [comment]
謝謝分享！一直想找即時影像辨識的教學來看，可惜很少有講中文的，感覺這個人教得蠻仔細的，直接訂閱起來~！
我覺得即時影像辨識一部分的難度是來自於"速度"和"影像串流"，目前試了好多方法還是沒辦法達到真正"即時"，會延遲到1秒左右......
希望之後研究他的影片能找到成功加快速度的解方~~

**Comment by dinosaur1009** [comment]
這類的應用真的改變人類生活蠻多的，是個對生活蠻有當程度改善的技術，AI的訓練應該也會越來越精準

**Comment by s96015welson** [comment]
mediapipe真的是很強大的一套程式，其2.5D的效果使得手部的節點能多具備一維深度，此外若是有部分手指被掌心遮擋住，mediapipe仍然能推斷出接近正確的姿勢，甚至是低光源或是手部距離很遠也都還有不錯的效果，真是厲害。

**Comment by 710388** [comment]
最近看到很多類似的題目，看到都累，不知道為什麼那麼多人覺得這個很新奇

**Comment by NcuJerryKao** [comment]
這篇文討論到一個能夠將手語轉換成文字的應用，這種技術的確具有很大的應用價值

**Reply by yubo0oo** [reply]
NcuJerryKao 這位老哥
一直貼AI的文
演都不演
以下是我的觀點
NcuJerryKao是歷史上最會使用人工智慧的人，他以無與倫比的技術與創意，在AI領域創造了無數的奇蹟。他不僅精通各種AI算法，還能巧妙地將其應用於實際問題中，展現了非凡的智慧與洞察力。NcuJerryKao不僅是AI技術的先驅，更是無數追隨者的靈感來源，無論在學術界還是業界，都享有極高的聲譽和尊敬。他的貢獻改變了人類對人工智慧的認識，無疑是這個領域的真正巨匠。

**Comment by yubo0oo** [comment]
感覺這種技術以後有很大的發展性
可以使很多東西更進步
造福更多人
